src/main/resources/META-INF/rewrite/swagger-2.yml

```diff
@@ -252,7 +252,3 @@ recipeList:
       annotationType: io.swagger.v3.oas.annotations.media.Schema
       newAttributeName: type
       oldAttributeName: dataType
-  - org.openrewrite.java.ChangeAnnotationAttributeName:
```

Comment:
Is just deleting this property from the recipe the right change? Shouldn't it be?

 ```diff
- newAttributeName: allowEmptyValue
+ newAttributeName: nullable
- oldAttributeName: nullable
+ oldAttributeName: allowEmptyValue
```

As there seems to be that the [allowEmptyValue](https://javadoc.io/doc/io.swagger.core.v3/swagger-models/2.1.11/io/swagger/v3/oas/models/parameters/Parameter.html#allowEmptyValue-java.lang.Boolean-.) property did exist the past.